presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestRewriteAggregationIfToFilter.java

```diff
@@ -402,4 +435,113 @@ public void testFireForOneOfTwoAggregations()
                                                         .build(),
                                                 values("ds")))));
     }
+
+    @Test
+    public void testArrayOffset()
+    {
+        for (String unwrapIf : new String[] {"true", "false"}) {
+            tester().assertThat(new RewriteAggregationIfToFilter(getFunctionManager()))
+                    .setSystemProperty(AGGREGATION_IF_TO_FILTER_REWRITE_ENABLED, "true")
+                    .setSystemProperty(AGGREGATION_IF_TO_FILTER_REWRITE_UNWRAP_IF_ENABLED, unwrapIf)
+                    .on(p -> {
+                        VariableReferenceExpression a = p.variable("a", new ArrayType(BIGINT));
+                        VariableReferenceExpression a1 = p.variable("a1", BIGINT);
+                        return p.aggregation(ap -> ap.globalGrouping().step(AggregationNode.Step.FINAL)
```

Comment:
Let's give these variable more meaningful names like variable1, variable2 or at least a1 and a2. Same for ap and other naming conventions in this test.

Reply:
Done.